src/desktop/apps/authentication/helpers.setCookies.ts

```diff
@@ -0,0 +1,15 @@
+import Cookies from "cookies-js"
+
+export const setCookies = options => {
```

Comment:
Why did this need to be moved out of the helpers file? This is a non-standard naming convention for Force.

Reply:
It was removed while you were cleaning up the logged out user, this is the only function being used from the helpers file in novo.